src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -39,6 +39,7 @@
         .protoVersionBound(packageConfig.protoVersionBound(language))
         .commonProtosVersionBound(packageConfig.commonProtosVersionBound(language))
         .authVersionBound(packageConfig.authVersionBound(language))
+        .packageName(packageConfig.packageName(language))
```

Comment:
Alternatively, would you prefer a `protoPackageName` method _in addition to_ the `gapic` and `grpc` equivalents? This gets around my concern.